### DISCUSSION TITLE
Animate

### DISCUSSION THREAD

**Discussion by dockinkong** [Megabit/Blazorise#3881] [Q&A]
Hello,

Maybe I missed something, but since release 1.0, animations seems to not be working anymore in my project...

Is there something to add or configure to get animations working like before?

Thanks for help

**Comment by stsrki** [comment]
We are working with charts v3 starting from Blazorise v1.0, Their API has changed in some ways.

Can you try this? https://github.com/Megabit/Blazorise/issues/3880#issuecomment-1153701384

**Reply by dockinkong** [reply]
Hello,
Thanks for you reply. I did charts updates to get it work and animations are ok.
My probleme is that a simple fade is not working anymore...

**Reply by dockinkong** [reply]
Blazorise.Animate is not loaded in _content

**Reply by stsrki** [reply]
I understood you want help with chart animations. My mistake.

---

Unlike other components, the Animate still requires defining JS files manually. You need to include these in your `index.html`.

![image](https://user-images.githubusercontent.com/900302/173344423-2a3351bc-a9b8-48e9-a97f-d9ecea48515f.png)

**Reply by dockinkong** [reply]
Oh ok, this was not the case before I think... Thanks a lot :)
